view/src/storage.rs

```diff
@@ -343,37 +399,40 @@ impl Storage {
 
         let nullifier_bytes = nullifier.0.to_bytes().to_vec();
```

Comment:
Compare this ....

Reply:
https://github.com/penumbra-zone/penumbra/pull/2372/files#r1170603076

Reply:
This turns out to be fine by accident, since both of these give the same array serialization.